website/code_issues.qmd

```diff
@@ -227,4 +227,34 @@ The issue described in the recipe, also applies to the use of other function whi
 
 --------------------------
 
+# Writing Files and Directories to the Home Filespace
+
+## Problem
+
+You are writing by default or in your examples, vignettes or tests in the user's home filespace, including the package directory and `getwd()`.
+
+## Solution 
+
+Omit any default path in your functions and write to `tempdir()` in examples, vignettes or tests.  
+
+
+### Details
+
+<!--adding this is a test for including the CRAN mail "template" for each recipe. This version is a small collapsible text box which I will add for (almost) every Issue. Maybe change the title of the boxes?--> 
+
+::: {.callout-note title="Comment of CRAN maintainers" appearance="simple" icon=false collapse=true}
+
+Please ensure that your functions do not write by default or in your examples/vignettes/tests in the user's home filespace (including the package directory and getwd()). This is not allowed by CRAN policies. Please omit any default path in writing functions. In your examples/vignettes/tests you can write to tempdir().
+
+:::
+
+The [CRAN Repository Policy](https://cran.r-project.org/web/packages/policies.html) states that:
```

Comment:
I guess that CRAN Repository Policy is linked a lot. 
With Markdown one can name a link and use that instead of repeating the link multiple times. Maybe have them on index.qmd and reuse across the book like [Bioconductor does](https://github.com/Bioconductor/pkgrevdocs/blob/devel/index.Rmd#L55), this way is easier to change if anything happens to the url and the text is shorter and cleaner in qmd.